src/main/java/IllegalCommandException.java

```diff
@@ -0,0 +1,2 @@
+public class IllegalCommandException extends Exception{
+}
```

Comment:
Empty Class. Please add at least a constructor with valid instantiations/assignments, so if this error is thrown, the description shown in correct.